README.md

```diff
@@ -276,6 +276,8 @@ The file `common.ts` with type [`AppConfig`](src/config/types.ts) contains impor
 - `tokenLists`: List of token lists including the uri and the parser name to be used to parse the token list. Please update the tokenParserMap in the `src/config/utils.ts` file to include the parser name and the parser function.
 - `sdk`:
   - `cacheTTL`: When the app loads, it will ignore any cached data if it is older than the cacheTTL time in milliseconds. If set to 0, the app will always ignore the cache data and fetch new data on load.
+  - `batchSize`: Amount of rpc request per batch
```

Comment:
I find it confusing "Amount of rpc request per batch" - it's actually how many blocks in a range when reading logs